src/explorer/defaultSettings.js

```diff
@@ -1,6 +1,7 @@
 const defaultSettings = {
   ignoredColumns: [],
   meta: [],
+  defaultCodebookSettings: {},
```

Comment:
effectively a single codebook settings object that applies to all codebooks?  rather than individual settings?

Reply:
Nailed it. 🔨